### PR TITLE
typo from case loop (RunB has ver1 and 2)

### PR DESCRIPTION
**Pull request by jeongsumin**

### CODE REVIEW

File: TopAnalysis/test/fcncTriLepton/01_prod_ntuple.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ ERA4HLT=$DATATYPE
 if [ ${DATATYPE::3} == "Run" ]; then
 # DATATYPE=${DATATYPE0::7} ## This gives Run2018A -> Run2018
   case $ERA in
-    Run2016B|Run2016C|Run2016D|Run2016E)
+    Run2016B_ver1|Run2016B_ver2|Run2016C|Run2016D|Run2016E)
```
Comment (jhgoh) [inline TopAnalysis/test/fcncTriLepton/01_prod_ntuple.sh]: My suggestions is:
```bash
case ${ERA::8} in
  Run2017B|Run2016C|Run2016D|Run2016E)
```

File: TopAnalysis/data/grouping/fcncTrilepton.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        title: "Data"
+        datasets:
+            - Run2016.MuonEG
+            - Run2016.SingleMuon
+            - Run2016.SingleElectron
+
+    DYJets:
+        title: "Z/#gamma^{*}+Jets#rightarrow l^{+}l^{-}"
+        weight: "puWeight*BtagWeight"
+        #weight: "genWeight*puWeight*BtagWeight"
```
Comment (jhgoh) [inline TopAnalysis/data/grouping/fcncTrilepton.yaml]: please use the "genWeight*puWeight*BtagWeight"

Comment (jeongsumin) [reply TopAnalysis/data/grouping/fcncTrilepton.yaml]: Do I use genWeight only at fcnc? or ttbb together?

Comment (jhgoh) [reply TopAnalysis/data/grouping/fcncTrilepton.yaml]: better together.

Comment (jeongsumin) [reply TopAnalysis/data/grouping/fcncTrilepton.yaml]: Ok, then I will do that with ttbb. Thanks!

**Pull request merged**